index.html

```diff
@@ -1,3 +1,70 @@
-<html>
-  
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>E-care</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+
+<body>
+
+    <nav>
+        <div class="nav">
+            <div class="nav-logo"><a href="#" class="logo">E-care</a></div>
+            <div class="nav-links">
+                <ul class="nav-list">
+                    <li class="nav-items"><a href="#" class="nav-link">Home</a></li>
+                    <li class="nav-items"><a href="#" class="nav-link">About</a></li>
+                    <li class="nav-items"><a href="#" class="nav-link">Services</a></li>
+                    <li class="nav-items"><a href="#" class="nav-link">Contact</a></li>
+                    <li class="nav-items"><a href="#" class="nav-link">Login</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <main class="main">
+
+        <header class="section-header">
+            <div>
+                <h1>Best healthcare <br> help center</h1>
+                <p>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat
```

Comment:
please come up with a short write-up here. remove the generated lorem ipsum write-up